tv_apps/browser/js/sync/tabList.js

```diff
@@ -200,33 +200,31 @@
       textEl.classList.add('text');
       titleEl.textContent = `${data.clientName} (${data.tabs.length})`;
 
-      if (lastUpdateTimeAgo.year) {
-        textEl.dataset.l10nId = 'fxsync-last-updated-more-than-a-year';
-        textEl.dataset.l10nArgs = JSON.stringify({
-          value: lastUpdateTimeAgo.year
-        });
-      } else if (lastUpdateTimeAgo.day) {
-        textEl.dataset.l10nId = 'fxsync-last-updated-more-than-a-day';
-        textEl.dataset.l10nArgs = JSON.stringify({
-          value: lastUpdateTimeAgo.day
-        });
-      } else if (lastUpdateTimeAgo.hour) {
-        textEl.dataset.l10nId = 'fxsync-last-updated-more-than-an-hour';
-        textEl.dataset.l10nArgs = JSON.stringify({
-          value: lastUpdateTimeAgo.hour
-        });
-      } else if (lastUpdateTimeAgo.minute) {
-        textEl.dataset.l10nId = 'fxsync-last-updated-more-than-a-minute';
-        textEl.dataset.l10nArgs = JSON.stringify({
-          value: lastUpdateTimeAgo.minute
-        });
-      } else if (lastUpdateTimeAgo.second) {
-        textEl.dataset.l10nId = 'fxsync-last-updated-more-than-a-second';
-        textEl.dataset.l10nArgs = JSON.stringify({
-          value: lastUpdateTimeAgo.second
-        });
+      let minute = Math.floor((Date.now() - data.timestamp) / (1000 * 60)),
+          hour = minute ? Math.floor(minute / 60) : 0,
+          day = hour ? Math.floor(hour / 24) : 0,
+          year = day ? Math.floor(day / 365) : 0,
+          timeUnit = '';
+
+      if (year) {
+        timeUnit = 'year';
+      } else if (day) {
+        timeUnit = 'day';
+      } else if (hour) {
+        timeUnit = 'hour';
+      } else {
+        timeUnit = 'minute';
       }
 
+      let rtf = new mozIntl.RelativeTimeFormat(navigator.languages, {
+        unit: timeUnit
+      });
```

Comment:
Instead of this, you can use `unit: 'bestFit'` and RelativeTimeFormat will choose the appropriate unit.